src/about.cpp

```diff
@@ -159,8 +162,11 @@ class c_AboutDialog: public wxDialog
 
 void c_AboutDialog::OnLibrariesClick(wxCommandEvent&)
 {
+    // glGetString() segfaults on macOS. I guess GL Context is not initialized.
```

Comment:
You're right, but this `#if` is needed anyway (in case OpenGL back end is disabled in `config.in`, there's a build error here). So I think we don't need this comment.

Reply:
The comment should have gone under the #if expression as it is irrelevant to the expression but a crash I have not resolved yet. I can remove the comment anyway and have a bug tix open instead.

Reply:
Yes, please do that (remove the comment, but open a bug). Now that I thought about it, the bug seems to be due to a) having OpenGL support built-in, b) launching ImPPG in CPU mode, so at no point was an OpenGL context initialized/made current, c) attempting to call a `gl*` function. Under Windows and Linux c) is allowed (just returns an empty/null string), but not on macOS.